package.json

```diff
@@ -10,19 +10,26 @@
     "build": "brunch build --production"
   },
   "dependencies": {
-    "react": "^15.4",
-    "react-dom": "^15.4",
-    "react-redux": "~4.3.0",
-    "redux": "~3.2.1"
+    "prop-types": "^15.6.0",
+    "react": "^16.2",
+    "react-dom": "^16.2",
+    "react-redux": "~5.0.6",
+    "redux": "~3.7.2"
   },
   "devDependencies": {
     "auto-reload-brunch": "^2",
-    "hmr-brunch": "^0.1",
-    "babel-brunch": "~6.0.0",
+    "babel-brunch": "~6.1.1",
+    "babel-preset-env": "^1.6.1",
```

Comment:
The deprecation warning needs to be solved in brunch itself, this is added here to be able to have an expected, controllable node environment behavior, we need to get rid of this at npm install phase:

```
npm WARN deprecated babel-preset-latest@6.24.1: We're super 😸  excited that you're trying to use ES2017+ syntax, but instead of making more yearly presets 😭 , Babel now has a better preset that we recommend you use instead: npm install babel-preset-env --save-dev. preset-env without options will compile ES2015+ down to ES5 just like using all the presets together and thus is more future proof. It also allows you to target specific browsers so that Babel can do less work and you can ship native ES2015+ to user 😎 ! We are also in the process of releasing v7, so please give http://babeljs.io/blog/2017/09/12/planning-for-7.0 a read and help test it out in beta! Thanks so much for using Babel 🙏, please give us a follow on Twitter @babeljs for news on Babel, join slack.babeljs.io for discussion/development and help support the project at opencollective.com/babel
```